library/std/src/hash/random.rs

```diff
@@ -2,9 +2,6 @@
 //! [`collections`] module without actually publicly exporting them, so that parts of that
 //! implementation can more easily be moved to the [`alloc`] crate.
```

Comment:
It looks like this comment is outdated too.

Reply:
You are right. So should this entire module be unmoduled and inlined into another file?

Reply:
Why not just inline this into https://github.com/rust-lang/rust/blob/master/library/std/src/hash/mod.rs without mod wrapper?